### PR TITLE
feat(an): Information screen data gathering

### PR DESCRIPTION
**Pull request by Thorrik58**
# Added Data gathering information screen

https://app.clickup.com/t/nz7nzn

## What

Add information screen for the data gathering

## Why

Needed for application

## Screenshots / Gifs

image

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: libs/application/templates/accident-notification/src/lib/messages/externalData.ts [hunk 1]
```diff
@@ -2,84 +2,105 @@ import { defineMessages } from 'react-intl'
...
-      id:
-        'an.application:section.backgroundInformation.externalData.sectionTitle',
+      id: 'an.application:section.agreementDescription.sectionTitle',
+      defaultMessage: 'Upplýsingar um meðferð á gögnum',
+      description: 'Information on data handling',
+    },
+    bulletOne: {
+      id: 'an.application:section.agreementDescription.BulletOne',
+      defaultMessage:
+        'Vinnueftirlit ríkisins fær afrit þessa eyðublaðs á grundvelli 4. mgr. 79. gr. laga nr. 46/1980. Tryggingastofnun ríkisins fær upplýsingar um bótagreiðslur sem ekki greiðast samhliða bótum stofnunarinnar, sbr. 14. gr. laga nr. 45/2015..',
```
Comment (bensijons) [inline libs/application/templates/accident-notification/src/lib/messages/externalData.ts:14]: This is just the default message so it maybe doesn't matter too much but there are two dots at the end here

File: libs/application/templates/accident-notification/src/lib/messages/externalData.ts [hunk 2]
```diff
@@ -2,84 +2,105 @@ import { defineMessages } from 'react-intl'
...
+      description: 'List item 1 on data gathering information',
+    },
+    bulletTwo: {
+      id: 'an.application:section.agreementDescription.BulletTwo',
+      defaultMessage:
+        'Þannig verður eitthvað óvænt að hafa átt sér stað og skilyrði er að óhappið verði ekki rakið til undirliggjandi sjúkdóms eða meinsemda hjá þeim sem fyrir óhappi verður..',
```
Comment (bensijons) [inline libs/application/templates/accident-notification/src/lib/messages/externalData.ts:20]: Same here

**Review by bensijons — APPROVED** [review]
LGTM just two very minor comments

File: libs/application/templates/accident-notification/src/lib/messages/externalData.ts (hunk 1 above)
Comment (Thorrik58) [reply libs/application/templates/accident-notification/src/lib/messages/externalData.ts:14]: Ill fix it in the next pr, was too quick to add automerge 😅

**Pull request merged**